src/components/JoinTeam/JoinTeam.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+import Header from '../Header/Header'
+
+function CreateTeam() {
+  const teamAccess = useSelector(store => store.teamAccess)
+  const [teamCode, setTeamCode] = useState('');
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  // useEffect to get all team id's and access codes
+  useEffect(() => {
+    dispatch({ type: 'FETCH_TEAM_ACCESS' });
+  }, [dispatch]);
+
+
+  const joinTeam = (event) => {
+    event.preventDefault();
+    for (let code of teamAccess) {
+      if (teamCode.toUpperCase() === code.accessCode) {
```

Comment:
Client side controls are not security 😉  Anyone could look at the chrome console, and see all the available access code 😱